modules/ossm-config-otel.adoc

```diff
@@ -146,5 +146,10 @@ $ oc exec -it -n bookinfo deployments/productpage-v1 -c istio-proxy -- curl loca
 +
 [source,terminal]
 ----
-$ kubectl get routes -n tempo tempo-sample-query-frontend
-----
\ No newline at end of file
+$ oc get routes -n tempo tempo-sample-query-frontend
```

Comment:
Technically out of scope for the PR, but if you're in here, I notice a handful of other command blocks in this procedure module that are missing their `$` prompt, and an instance of `<revision-name>` that should be `<revision_name>`.

Reply:
I noticed a few were missing while updating, no idea how it made it through peer review and merge review the first time! :laughing: 

I was adding them in as I noticed them. Seems I missed some. Thanks!

Reply:
<revision-name> is correct, see

oc label namespace <namespace-name> istio.io/rev=<revision-name> here: https://83665--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/update/ossm-updating-openshift-service-mesh